### PR TITLE
Add skeleton

### PR DESCRIPTION
**Pull request by BasitKorede**
# Add Capstone Project

This pull request adds the following:

- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
- The mobile menu should have a close (X) button that closes the menu.
- Dynamic page

  - The section "Featured speakers" should be created dynamically in JavaScript.
  You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for 
     this section dynamically.
### [Loom video](https://www.loom.com/share/d111c2f146b84366b10ed0f88fb20aaa?sharedAppSource=personal_library)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# MY PROFESSIONAL PORTFOLIO
+# REFUGEE FOOD FESTIVAL (Html/css capstone)
```
Comment (MahmoudBakr23) [inline README.md:3]: - [ ] Kindly fill the empty sections in your README file and delete the unnecessary ones in order to have a completely professional repo with a nicer look 🙏 😄 
- [ ] Please add acknowledgement to the design author in your README file

File: about.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  <body class="about-body">
+    <header class="d-flex j-content-btw nav-bar">
+      <div class="name-div">
+        <a href="index.html" class="name-anchor">
+          <img
+            class="project-logo"
+            src="./images/restuarant.logo.png"
+            alt="page logo"
+        /></a>
+      </div>
```
Comment (MahmoudBakr23) [inline about.html:23]: - [ ] Please follow the given design wireframe in order to practice the concept of transforming code into a design in the exact same look. This is a very important chance to start improving such skills as a developer 😉
_Kindly consider all of the following design comments with the same importance._ 🙏
- [ ] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your header section on both desktop and mobile version

File: about.html [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+            Systems Summit activities and efforts to transform food systems can
+            be leveraged for greater food security, better nutrition, and more
+            viable livelihoods for refugees.
+          </p>
+
+          <p class="about-second-paragraph text-align-center">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam
+            elementum risus
+          </p>
+        </div>
```
Comment (MahmoudBakr23) [inline about.html]: - [ ] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your main section on both desktop and mobile version

File: index.html [hunk 4]
```diff
@@ -4,13 +4,234 @@
...
+                  src="./images/speaking-summit.png"
+                  alt="speaker"
+                />
+              </li>
+              <li class="main-program-ul-li topic">Lecture</li>
+              <li class="main-program-ul-li">
+                listen to the speaker from various country about sharing and
+                opening
+              </li>
+            </ul>
```
Comment (MahmoudBakr23) [inline index.html:117]: - [x] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your program section on both desktop and mobile version

File: mobile-menu.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  menuButton.classList.toggle('display-none')
+  bodyElement.style.overflow = 'hidden'
+})
+
+menuButton.addEventListener('click', () => {
+  navbarLinks.classList.toggle('display-none')
+  toggleButton.classList.toggle('display-none')
+  menuButton.classList.toggle('display-none')
+  bodyElement.style.overflow = 'auto'
+})
```
Comment (MahmoudBakr23) [inline mobile-menu.js]: - [ ] Please make sure to fix the menu button when it's changing from the mobile menu to the desktop, it still appears on the page.

File: index.html [hunk 6]
```diff
@@ -4,13 +4,234 @@
...
+            </li>
+
+            <li class="daumakakau">
+              <a><p>daumakakau</p> </a>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </footer>
+    <script src="mobile-menu.js" defer></script>
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] - [ ] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your partners and footer sections on both desktop and mobile version

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @BasitKorede 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linters are correct
- Great use of javascript to set up the objects of the website

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (BasitKorede) [reply index.html]: Thanks for the review change made

File: mobile-menu.js (hunk 5 above)
Comment (BasitKorede) [reply mobile-menu.js]: Thanks for the review change made

File: index.html (hunk 4 above)
Comment (BasitKorede) [reply index.html:117]: Thanks for the review change made

File: about.html (hunk 2 above)
Comment (BasitKorede) [reply about.html:23]: Thanks for the review change made

File: index.html (hunk 4 above)
Comment (BasitKorede) [reply index.html:117]: Thanks for the review change made

**Review by Gambit142 — COMMENTED** [review]
Hi @BasitKorede 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that GitHub actions did not check for linters error. Kindly fix it and ensure Github actions check for linters before submitting for a review again. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by BasitKorede** [comment]
Thanks for your generosity changes made

File: README.md (hunk 1 above)
Comment (keroloussamy) [inline README.md:3]: - [ ] Kindly as the previous review asked, add the "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Here your section is not mention that
image

File: css/style.css [hunk 7]
```diff
@@ -1,9 +1,379 @@
...
+  /* background-image: url(../images/refugee.jpg); */
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.home-page-first-div {
+  padding: 14px;
+  margin-bottom: 95px;
+}
```
Comment (keroloussamy) [inline css/style.css]: - [ ] Kindly as asked by the previous reviewer, Your first section doesn't look like the Original design. You can just make the `padding` bigger ex `padding: 140px;`.

Your design
image

Original design 
image

File: css/style.css [hunk 8]
```diff
@@ -1,9 +1,379 @@
...
+  margin: 0 auto;
+}
+
+.second-footer-paragraph {
+  margin: 24px 0;
+  line-height: 23px;
+}
+
+.partner-header {
+  color: #fff;
```
Comment (keroloussamy) [inline css/style.css:381]: - [ ] Kindly to match the previous review changes, you can add `padding-top: 30px` to make your design match the original design.

File: css/style.css [hunk 9]
```diff
@@ -1,9 +1,379 @@
...
+  margin: 24px 0;
+}
+
+.join-anchore:hover {
+  opacity: 0.8;
+}
+
+.all-wrappers {
+  margin: 25px 0;
+}
```
Comment (keroloussamy) [inline css/style.css]: - [ ] Kindly to match the previous review changes, you can remove the `margin: 25px 0;` to make your design match the original design.

File: css/style.css [hunk 10]
```diff
@@ -1,9 +1,379 @@
...
+
+.footer-image-title {
+  font-size: 17px;
+}
+
+.footer-ul {
+  width: 90%;
+  margin: 4px auto;
+}
```
Comment (keroloussamy) [inline css/style.css]: - [ ] Kindly to match the previous review changes, you can add `padding-bottom: 40px;` to make your design match the original design.

File: css/style.css [hunk 11]
```diff
@@ -1,9 +1,379 @@
...
+
+.footer {
+  background-color: #fff;
+}
```
Comment (keroloussamy) [inline css/style.css]: - [ ] Kindly to match the previous review changes, you can add `padding: 15px;` to make your design match the original design.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @BasitKorede, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 7 above)
Comment (BasitKorede) [reply css/style.css]: Thanks for the review changes made

File: css/style.css (hunk 11 above)
Comment (BasitKorede) [reply css/style.css]: Thanks for the review

File: css/style.css (hunk 8 above)
Comment (BasitKorede) [reply css/style.css:381]: Thanks for your review

File: README.md (hunk 1 above)
Comment (mateo951) [reply README.md:3]: Addressed ✔

File: css/style.css (hunk 11 above)
Comment (mateo951) [reply css/style.css]: Addressed ✔

File: css/style.css (hunk 10 above)
Comment (mateo951) [reply css/style.css]: Addressed ✔

File: css/style.css (hunk 9 above)
Comment (mateo951) [reply css/style.css]: Addressed ✔

File: css/style.css (hunk 8 above)
Comment (mateo951) [reply css/style.css:381]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @BasitKorede, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**